.travis.yml

```diff
@@ -1,6 +1,7 @@
-sudo: false
+---
 language: ruby
+cache: bundler
+before_install: gem install bundler -v 2.1.4
 rvm:
-  - 2.7.0
+  - 2.7.1
   - ruby-head
```

Comment:
I am also in favor of adding 2.5.x and 2.6.x support.
D'you think we can (or we should) do that?

Reply:
Something like:
```
rvm:
  - 2.5.8
  - 2.6.6
  - 2.7.1
  - ruby-head
```

Reply:
Sounds good to me 👍